src/ResourceManager/Network/Commands.Network/NetworkWatcher/PacketCapture/StopAzureNetworkWatcherPacketCaptureCommand.cs

```diff
@@ -0,0 +1,67 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Network.Models;
+using Microsoft.Azure.Management.Network;
+using System.Collections.Generic;
+using System.Management.Automation;
+using MNM = Microsoft.Azure.Management.Network.Models;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet(VerbsLifecycle.Stop, "AzureRmNetworkWatcherPacketCapture", SupportsShouldProcess = true, DefaultParameterSetName = "SetByResource"), OutputType(typeof(bool))]
+    public class StopAzureNetworkWatcherPacketCaptureCommand : PacketCaptureBaseCmdlet
+    {
+        [Parameter(
+             Mandatory = true,
+             ValueFromPipeline = true,
+             HelpMessage = "The network watcher resource.",
+             ParameterSetName = "SetByResource")]
+        public PSNetworkWatcher NetworkWatcher { get; set; }
+
+        [Alias("Name")]
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
+            HelpMessage = "The name of network watcher.",
+            ParameterSetName = "SetByName")]
+        public string NetworkWatcherName { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipeline = true,
+            HelpMessage = "The name of the network watcher resource group.",
+            ParameterSetName = "SetByName")]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "The packet capture name.")]
+        [ValidateNotNullOrEmpty]
+        public virtual string PacketCaptureName { get; set; }
+
+        public override void Execute()
+        {
+            base.Execute();
+            if (ParameterSetName.Contains("SetByResource"))
```

Comment:
@irrogozh this cmdlet implements `SupportsShouldProcess`, but there is no `ConfirmAction` method wrapping these calls